content/source/docs/cloud/integrations/service-now/example-customizations.html.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: "cloud"
-page_title: "Example Customizations - ServiceNow Service Catalog Integration - Terraform Cloud"
+page_title: "Example Customizations - ServiceNow Service Catalog Integration - Terraform Cloud and Terraform Enterprise"
```

Comment:
I don't think ServiceNow is meant to be used with Cloud, although I don't remember why exactly.

Reply:
Same comment for all the Service Now pages

Reply:
For that one, I think I'll just leave it like it is in the PR for now, since we'll be adjusting those docs at some point soon anyway. (ServiceNow can work with TFC too, but if you're at the scale where you need SN in the first place, you're probably an enterprise customer by default... is mostly what that's all about.)